src/minichain/transaction.py

```
@@ -0,0 +1,144 @@
+        if self.is_coinbase():
+            return True
+        if not self._validate_common_fields():
+            return False
+        if not _is_lower_hex(self.public_key, PUBLIC_KEY_HEX_LENGTH):
+            return False
+        if not _is_lower_hex(self.signature, SIGNATURE_HEX_LENGTH):
+            return False
+
+        try:
+            verify_key = deserialize_verify_key(self.public_key)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)